mon-pix/config/environment.js

```diff
@@ -198,8 +198,15 @@ module.exports = function (environment) {
   if (environment === 'test') {
     // Testem prefers this...
     ENV.locationType = 'none';
+    ENV.APP.APPLICATION_NAME = 'pix-app-local';
+    ENV.APP.APP_VERSION = 'development';
```

Comment:
Est-ce que redéfinir cette variable d’env est vraiment nécessaire ? Je pense que non. Mais si on voulait alors ne faudrait-il pas plutôt mettre le code suivant ? :
```suggestion
    ENV.APP.APP_VERSION = 'test';
```

Reply:
Tout à fait d'accord pour préciser 'test' au lieu de 'development'. 
Sur la pertinence de déclarer cette variable, c'est la même démarche que pour `ENV.APP.APPLICATION_NAME = 'pix-app-local';` pour ma part.

Reply:
Finalement, j'ai enlevé cette déclaration. En effet, des tests unitaires vérifiaient que le header `X-App-Version` contient bien la mention 'development'. Et je ne veux pas modifier des tests existants dans cette PR.